Using backend: pytorch
Traceback (most recent call last):
  File "/home/cresswellclayec/network/predict_e2e.py", line 324, in <module>
    pred.predict(args.a3m_fn, args.out_prefix, args.hhr, args.atab)
  File "/home/cresswellclayec/network/predict_e2e.py", line 116, in predict
    msa = parse_a3m(a3m_fn)
  File "/spin1/home/linux/cresswellclayec/network/parsers.py", line 26, in parse_a3m
    for line in open(filename,"r"):
FileNotFoundError: [Errno 2] No such file or directory: '/gpfs/gsfs8/users/cresswellclayec/DCA_ER/furano_periwal_prot/rosettafold/e2e_out_vp0_p14/t000_.msa0.a3m'
